docs/outlook/internet-headers.md

```diff
@@ -1,15 +1,19 @@
 ---
 title: Get and set internet headers
 description: How to get and set internet headers on a message in an Outlook add-in.
-ms.date: 11/04/2019
+ms.date: 04/10/2019
 localization_priority: Normal
 ---
 
 # Get and set internet headers on a message in an Outlook add-in
 
 ## Background
 
-A common requirement in Outlook add-ins development is to store custom properties associated with an add-in at different levels. At present, custom properties are stored at the item or mailbox level.
+A common requirement in Outlook add-ins development is to store custom properties associated with an add-in at different levels. 
+
+![Diagram of internet headers. Text: User 1 sends email. Add-in manages custom internet headers while user is composing email. User 2 receives the email. Add-in gets internet headers from received email then parses and uses custom headers. ](../images/outlook-internet-headers.png)
```

Comment:
The original location in the first commit is correct.

Reply:
Yeah I debated about where to put it, I think I will move it back to where I had it the first time.